src/ceph_fuse.cc

```diff
@@ -95,6 +95,9 @@ int main(int argc, const char **argv, const char *envp[]) {
     } else if (ceph_argparse_flag(args, i, "--localize-reads", (char*)nullptr)) {
       cerr << "setting CEPH_OSD_FLAG_LOCALIZE_READS" << std::endl;
       filer_flags |= CEPH_OSD_FLAG_LOCALIZE_READS;
+      if(end == std::next(i, 1)) {
+	break;
+      }
```

Comment:
As per the original author, it is correct that `--localize-reads` is suppose to set the flag and proceed with processing the other options, if any of them are available. This works fine normally. But the segmentation fault happens because the [erase](https://github.com/ceph/ceph/blob/9b72808469f7b3df9ff660279a427b94b1636b7d/src/common/ceph_argparse.cc#L285) is getting called second time with the same `i` when `--localize-reads` appears as the last argument. 

I would like to test this fix again today and check if any better way to handle this case. So putting DNM for now. Thanks.